ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/PartyAllocationResult.scala

```diff
@@ -16,19 +16,30 @@ object PartyAllocationResult {
     override def description: String = "Party successfully allocated"
   }
 
+  /** The system is overloaded, clients should back off exponentially */
+  final case object Overloaded extends PartyAllocationResult {
```

Comment:
```suggestion
  case object Overloaded extends PartyAllocationResult {
```
`final` is redundant on `object`s.